expr_checker.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import ast
+from inspect import getsource
+from textwrap import dedent
+
+
+def ast_default_check_type(method, value):
+    safe_type = (str, int, type(None), type(range(0)))
+
+    if type(value) not in safe_type and not (hasattr(value, "__self__") and type(value.__self__) in safe_type):
```

Comment:
```suggestion
    if type(value) not in safe_type:
```

remove this, and add `bound method` to the safe types list